### DISCUSSION TITLE
Using Catalytic for Search and Rescue Incident Management Application

### DISCUSSION THREAD

**Discussion by HeroesLament** [Jasperav/Catalytic#32] [General]
Hello Jasper! 

I just wanted to let you know, I'm currently drafting out my architecture for a revised Incident Management application for my local SAR Team. I've opted for Cassandra at the moment for its long-term stability and high-availability of the cluster, although I may just go straight for ScyllaDB as I progress. Does Cataytic currently have API parity for Cassandra in addition to Scylla?

Thanks!

**Comment by Jasperav** [comment]
Hey @HeroesLament, thanks for considering Catalytic for your project! Yes, this crate works perfectly fine with Cassandra and is well maintained. Although I don’t know your requirements, I would just go with Scylla, they are really stable and have customers with extremely high data sets.
